test/parallel/test-internal-webidl.js

```diff
@@ -0,0 +1,35 @@
+// Flags: --expose-internals
+'use strict';
+
+require('../common');
+const { describe, it } = require('node:test');
+const assert = require('node:assert');
+const { createInterfaceConverter } = require('internal/webidl');
+
+describe('internal/webidl', () => {
```

Comment:
nit: we shouldn't encourage writing tests that use webidl directly. It's harder to replicate when comparing other runtimes, than if we check if AbortSignal.any([null]) throws an error, for example

Reply:
Sounds good. I can drop this file. The other test file has covered this.